development-guidelines.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+---
+
+# Development Guidelines
+
```

Comment:
Can you add a qualifier here?  These are great guidelines, but they are very AllReady specific. 

Something like:  

> These guidelines are based on the experience with the AllReady project. These have been very helpful for us on this project, but may not be applicable to every project. We'll be updated these as we onboard other projects.

/cc @tonysurma